vcr/issuer/issuer.go

```diff
@@ -357,3 +424,52 @@ func (i issuer) isRevoked(credentialID ssi.URI) (bool, error) {
 func (i issuer) SearchCredential(credentialType ssi.URI, issuer did.DID, subject *ssi.URI) ([]vc.VerifiableCredential, error) {
 	return i.store.SearchCredential(credentialType, issuer, subject)
 }
+
+// statusListValidity is default validity of a statuslist credential. It should be
+const statusListValidity = 24 * time.Hour
+
+var statusList2021ContextURI = ssi.MustParseURI(jsonld.W3cStatusList2021Context)
+var statusList2021CredentialTypeURI = ssi.MustParseURI(credential.StatusList2021CredentialType)
+
+func (i issuer) StatusList(ctx context.Context, issuer did.DID, page string) (*vc.VerifiableCredential, error) {
+	// todo: get cached credential if available
+
+	// issuer
+	issuerAsSSI, err := ssi.ParseURI(issuer.String())
+	if err != nil {
+		return nil, err
+	}
+
+	// get credential subject
+	credSubject, err := i.statusListStore.CredentialSubject(ctx, issuer, page)
+	if err != nil {
+		return nil, err
+	}
+
+	// create statuslist credential template
+	iss := TimeFunc()
+	exp := iss.Add(statusListValidity)
+	template := vc.VerifiableCredential{
```

Comment:
do you have an example output VC which I can check (e.g. in the JSON-LD playground)

Reply:
below is the produced credential, does that work?
```
{
  "@context": [
    "https://www.w3.org/2018/credentials/v1",
    "https://w3id.org/vc/status-list/2021/v1",
    "https://w3c-ccg.github.io/lds-jws2020/contexts/lds-jws2020-v1.json"
  ],
  "credentialSubject": {
    "encodedList": "H4sIAAAAAAAA_-zAMQEAAADCIPunNsYeGAAAAAAAAAAAAAAAAAAAAJB7AAAA__-G0lSrAEAAAA==",
    "id": "https://example.com/iam/123/status/1",
    "statusPurpose": "revocation",
    "type": "StatusList2021"
  },
  "expirationDate": "2024-01-25T14:42:51.570276+01:00",
  "id": "did:web:example.com:iam:123#7e49434e-c41e-41d8-9722-11a22b776ee1",
  "issuanceDate": "2024-01-24T14:42:51.570276+01:00",
  "issuer": "did:web:example.com:iam:123",
  "proof": {
    "created": "2024-01-24T14:42:51.570276+01:00",
    "jws": "eyJhbGciOiJFUzI1NiIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il0sImtpZCI6ImRpZDp3ZWI6ZXhhbXBsZS5jb206aWFtOjEyMyNhYmMifQ..4ImqoHBsTtvyHP6i2Z2zkhBSJak-ObenEDCoxquGzMsSjZ0oi0o5JtriHYBt7VPV9H_Mm1HaRaHaW65VyiymKw",
    "proofPurpose": "assertionMethod",
    "type": "JsonWebSignature2020",
    "verificationMethod": "did:web:example.com:iam:123#abc"
  },
  "type": [
    "StatusList2021Credential",
    "VerifiableCredential"
  ]
}
```

Reply:
Yes, thanks,. We should probably package the JSON-LD context in the Nuts node?

`https://w3id.org/vc/status-list/2021/v1`

Reply:
already added as part of the read operations for statuslist